core/components/organisms/verticalNav/__stories__/CustomOptionsRenderer.story.jsx

```diff
@@ -35,7 +35,7 @@ export const CustomOptionsRenderer = () => {
         {
           name: 'care_management.tasks',
           label: 'Tasks',
-          icon: 'alarm',
+          icon: 'alarm_filled',
```

Comment:
Please check this once icon are not coming with filled value, is something got changed as in mds.innoavccer.com it is working fine but in this PR storybook the icon filled state is breaking. please check and verify this once attaching the screenshot for reference.

![Screenshot 2025-05-07 at 1 40 55 PM](https://github.com/user-attachments/assets/daf192d6-09aa-421c-a696-74c879af1686)
![Screenshot 2025-05-07 at 1 41 02 PM](https://github.com/user-attachments/assets/d726746e-4ffa-421a-ad83-fcfcd50b6536)